recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@
 # Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
 #
 # Distribution of the content approved by NVIDIA ( http://nvbugs/3052604 )
-{% set major_minor = "11.0" %}
+{% set major_minor = major_minor_ver %}
```

Comment:
You might have to set a default here for early stages of rendering.

Reply:
Good point. Thanks Isuru! 😄

Reply:
Added a `default` here ( https://github.com/conda-forge/cudatoolkit-feedstock/pull/31#pullrequestreview-526539482 ).